examples/replica_exchange/process_replica_exchange.py

```diff
@@ -0,0 +1,37 @@
+#!/Users/mrshirts/anaconda3/bin/python
+
+import os
+import pdb
+from simtk import unit
+from foldamers.cg_model.cgmodel import CGModel
+from foldamers.parameters.reweight import get_temperature_list
+from cg_openmm.simulation.rep_exch import *
+import numpy as np
+import pickle
+
+###
+#
+# This example demonstrates how to run a Yank replica exchange simulation
+# using a "CGModel" object built with the 'foldamers' software package.
+#
+###
+
+# Job settings
+output_directory = "output"
+if not os.path.exists(output_directory):
+    os.mkdir(output_directory)
+output_data = os.path.join(output_directory, "output.nc")
+
+# Replica exchange simulation settings
```

Comment:
Do these min and max temps need to be defined here?

Reply:
No, passed in the pkl for now.